.github/ISSUE_TEMPLATE/20-docs-issue.md

```diff
@@ -0,0 +1,46 @@
+Thank you for submitting a TFX documentation issue. Per our GitHub
+policy, we only address code/doc bugs, performance issues, feature requests, and
+build/installation issues on GitHub.
+
+The TFX docs are open source! To get involved, read the documentation
+contributor guide: https://www.tensorflow.org/community/contribute/docs
+
+## URL(s) with the issue:
+
+Please provide a link to the documentation entry, for example:
+https://www.tensorflow.org/tfx/guide
+
+## Description of issue (what needs changing):
+
+### Clear description: 
+
+For example, why should someone use this method? How is it useful?
+
+### Correct links
+
+Is the link to the source code correct?
+
+### Parameters defined
+
+Are all parameters defined and formatted correctly?
+
+### Returns defined
+
+Are return values defined?
+
+### Raises listed and defined
+
+Are the errors defined? For example,
+
+https://www.tensorflow.org/tfx/guide/examplegen
+
+### Usage example
+
+Is there a usage example?
+
+See the API guide: https://www.tensorflow.org/community/contribute/docs_ref
+on how to write testable usage examples.
```

Comment:
We don't actually do this ourselves yet, so maybe leave it out.

@rcrowe-google : we should discuss with TensorFlow doc team to better understand this and see if it makes sense for us to scope a change to this. My understanding to the benefit and cost here is limited right now.

Reply:
ping again? Can we remove the testable usage example for now since we don't it ourselves?